`yarn start:contract` fails: TypeError#1: Cannot delete property 'dispose' of function Symbol

Running the `yarn start:contract` command returns the following error:

```
agoric@AgoricVM:~/Desktop/demo$ yarn start:contract
yarn run v1.22.21
$ cd contract && yarn start
$ yarn docker:make clean start-contract print-key
$ docker compose exec agd make -C /workspace/contract clean start-contract print-key
make: Entering directory '/workspace/contract'
./scripts/build-proposal.sh
Removing intrinsics.Symbol.dispose
Removing intrinsics.Symbol.asyncDispose
agoric: run: running /workspace/contract/scripts/build-game1-start.js
start-game1-proposal.js module evaluating
SES Lockdown using options from environment variables "LOCKDOWN_OVERRIDE_TAMING"
Removing intrinsics.Symbol.dispose
failed to delete intrinsics.Symbol.dispose (TypeError#1)
TypeError#1: Cannot delete property 'dispose' of function Symbol() { [native code] }

(TypeError#1)
agoric: (Error#1)
Error#1: bundle-source failed with status 255
  at makeError (file:///usr/src/agoric-sdk/node_modules/ses/src/error/assert.js:243:17)
  at fail (file:///usr/src/agoric-sdk/node_modules/ses/src/error/assert.js:357:20)
  at Fail (file:///usr/src/agoric-sdk/node_modules/ses/src/error/assert.js:366:39)
  at createBundlesFromAbsolute (file:///workspace/node_modules/@agoric/internal/src/node/createBundles.js:41:11)
  at createBundles (file:///workspace/node_modules/@agoric/internal/src/node/createBundles.js:50:10)
  at getBundle (file:///workspace/node_modules/@agoric/deploy-script-support/src/writeCoreProposal.js:60:13)
  at install (file:///workspace/node_modules/@agoric/deploy-script-support/src/writeCoreProposal.js:67:22)
  at game1ProposalBuilder (file:///workspace/contract/scripts/build-game1-start.js:19:11)
  at writeCoreProposal (file:///workspace/node_modules/@agoric/deploy-script-support/src/writeCoreProposal.js:90:14)
  at default (file:///workspace/contract/scripts/build-game1-start.js:33:9)
  at async file:///usr/src/agoric-sdk/packages/agoric-cli/src/scripts.js:188:7
  at async Command.parseAsync (/usr/src/agoric-sdk/packages/agoric-cli/node_modules/commander/lib/command.js:935:5)
  at async main (file:///usr/src/agoric-sdk/packages/agoric-cli/src/main.js:400:5)

file:///workspace/contract/scripts/parseProposals.mjs:6
  throw Error(String.raw(template, ...args.map(val => String(val))));
        ^

Error: No swingset-core-eval found in proposal output: bundle-source --to /workspace/contract/bundles /workspace/contract/src/gameAssetContract.js game1

    at Fail (file:///workspace/contract/scripts/parseProposals.mjs:6:9)
    at parseProposalParts (file:///workspace/contract/scripts/parseProposals.mjs:24:23)
    at main (file:///workspace/contract/scripts/parseProposals.mjs:36:17)
    at file:///workspace/contract/scripts/parseProposals.mjs:41:1
    at ModuleJob.run (node:internal/modules/esm/module_job:195:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:336:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:106:12)

Node.js v18.19.0
./scripts/build-proposal.sh
Removing intrinsics.Symbol.dispose
Removing intrinsics.Symbol.asyncDispose
agoric: run: running /workspace/contract/scripts/build-game1-start.js
start-game1-proposal.js module evaluating
SES Lockdown using options from environment variables "LOCKDOWN_OVERRIDE_TAMING"
Removing intrinsics.Symbol.dispose
failed to delete intrinsics.Symbol.dispose (TypeError#1)
TypeError#1: Cannot delete property 'dispose' of function Symbol() { [native code] }

(TypeError#1)
agoric: (Error#1)
Error#1: bundle-source failed with status 255
  at makeError (file:///usr/src/agoric-sdk/node_modules/ses/src/error/assert.js:243:17)
  at fail (file:///usr/src/agoric-sdk/node_modules/ses/src/error/assert.js:357:20)
  at Fail (file:///usr/src/agoric-sdk/node_modules/ses/src/error/assert.js:366:39)
  at createBundlesFromAbsolute (file:///workspace/node_modules/@agoric/internal/src/node/createBundles.js:41:11)
  at createBundles (file:///workspace/node_modules/@agoric/internal/src/node/createBundles.js:50:10)
  at getBundle (file:///workspace/node_modules/@agoric/deploy-script-support/src/writeCoreProposal.js:60:13)
  at install (file:///workspace/node_modules/@agoric/deploy-script-support/src/writeCoreProposal.js:67:22)
  at game1ProposalBuilder (file:///workspace/contract/scripts/build-game1-start.js:19:11)
  at writeCoreProposal (file:///workspace/node_modules/@agoric/deploy-script-support/src/writeCoreProposal.js:90:14)
  at default (file:///workspace/contract/scripts/build-game1-start.js:33:9)
  at async file:///usr/src/agoric-sdk/packages/agoric-cli/src/scripts.js:188:7
  at async Command.parseAsync (/usr/src/agoric-sdk/packages/agoric-cli/node_modules/commander/lib/command.js:935:5)
  at async main (file:///usr/src/agoric-sdk/packages/agoric-cli/src/main.js:400:5)

file:///workspace/contract/scripts/parseProposals.mjs:6
  throw Error(String.raw(template, ...args.map(val => String(val))));
        ^

Error: No swingset-core-eval found in proposal output: bundle-source --to /workspace/contract/bundles /workspace/contract/src/gameAssetContract.js game1

    at Fail (file:///workspace/contract/scripts/parseProposals.mjs:6:9)
    at parseProposalParts (file:///workspace/contract/scripts/parseProposals.mjs:24:23)
    at main (file:///workspace/contract/scripts/parseProposals.mjs:36:17)
    at file:///workspace/contract/scripts/parseProposals.mjs:41:1
    at ModuleJob.run (node:internal/modules/esm/module_job:195:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:336:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:106:12)

Node.js v18.19.0
./scripts/install-bundles.sh
+ cd /workspace/contract
+ '[' -s bundles/bundle-list ']'
+ exit 1
make: *** [Makefile:110: install-bundles] Error 1
make: Leaving directory '/workspace/contract'
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

--- a/package.json
+++ b/package.json
@@ -10,7 +10,7 @@
   ],
   "resolutions-note": "work-around for https://github.com/Agoric/agoric-sdk/issues/8621",
   "resolutions": {
-    "ses": "0.18.4",
+    "ses": "0.18.5",
     "@endo/bundle-source": "2.5.2-upstream-rollup",
     "@endo/captp": "3.1.1",
     "@endo/compartment-mapper": "0.8.4",

